src/components/Cart/Summary/Buttons.vue

```diff
@@ -14,22 +14,22 @@
             class="np-cart__buttons__button"
             @click="resetOrder">
 
-      {{ $t('addOrder') }}
+      {{ $t("addOrder") }}
```

Comment:
Check your configuration, probably litner is changing your single qoute to double